spec/views/user_show_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+RSpec.describe 'User show', type: :feature do
+  before(:each) do
+    @user1 = User.create(
+      name: 'Tracey',
+      bio: 'I am a remote software developer',
+      photo: 'http://www.example.com/photo.jpg',
+      posts_counter: 6
+    )
+
+    @post = Post.create(
+      user: @user1,
+      title: 'My first post',
+      text: 'This is my first post',
+      comments_counter: 2,
+      likes_counter: 9
+    )
+
+    @post2 = Post.create(
+      user: @user1,
+      title: 'My second post',
+      text: 'This is my second post',
+      comments_counter: 3,
+      likes_counter: 10
+    )
+  end
+
+  describe 'GET /users/:id' do
+    it 'displays the correct username' do
+      visit user_path(@user1)
+
+      expect(page).to have_content('Tracey')
+    end
+
+    it 'displays the correct photo' do
+      visit user_path(@user1)
+
+      expect(page).to have_css('img[src="http://www.example.com/photo.jpg"]')
+    end
+
```

Comment:
- [ ] Kindly notice that you should make a test that tests that asserts you can see the number of posts the user has written.